modules/images/webp-uploads/load.php

```diff
@@ -663,9 +663,9 @@ function webp_uploads_wepb_fallback() {
 
 		i = d.createElement( i );
 		i.src = p + 'jIAAABXRUJQVlA4ICYAAACyAgCdASoCAAEALmk0mk0iIiIiIgBoSygABc6zbAAA/v56QAAAAA==';
-		i.onload = function() {
+		i.addEventListener( 'load', function() {
 			i.src = p + 'h4AAABXRUJQVlA4TBEAAAAvAQAAAAfQ//73v/+BiOh/AAA=';
-		};
+		}, { once: true } );
```

Comment:
This will not work in older browsers, I have created another PR that fixes this problem: #434. @mukeshpanchal27 could you please revert your changes and just use the same approach as in my PR? I'll close my PR as duplicate for this one.

Reply:
Thank you! PR updated with changes